pyaedt/modules/SolveSetup.py

```diff
@@ -1984,7 +1984,7 @@ def _get_net_names(self, app, file_fullname):
                 try:
                     net = net.replace("-", "m")
                     net = net.replace("+", "p")
-                    net_name = re.sub("[^a-zA-Z0-9 \n\.]", "_", net)
+                    net_name = re.sub("[^a-zA-Z0-9 .\n]", "_", net)
```

Comment:
Is this change safe? @SMoraisAnsys

Reply:
From my understanding, these code line is supposed to replace anything that is not a letter, digit, space, dot or breaking line with an underscore. As far I know, since we are using a character class, `[ ... ]`, this should work as . loses its special meaning outsid of such class and should behave as a literal dot.

Local result:
```python
>>> import re
>>> net = "is.this.is.sparta?"
>>> net_name = re.sub("[^a-zA-Z0-9 .\n]", "_", net)
>>> print(net_name)
is.this.is.sparta_
```